metamorph/processor.go

```diff
@@ -387,21 +387,22 @@ func (p *Processor) SendStatusMinedForTransaction(hash *chainhash.Hash, blockHas
 			resp.Close()
 			p.ProcessorResponseMap.Delete(hash)
 
-			if p.cbChannel != nil {
-				data, _ := p.store.Get(spanCtx, hash[:])
-
-				if data != nil && data.CallbackUrl != "" {
-					p.cbChannel <- &callbacker_api.Callback{
-						Hash:        data.Hash[:],
-						Url:         data.CallbackUrl,
-						Token:       data.CallbackToken,
-						Status:      int32(data.Status),
-						BlockHash:   data.BlockHash[:],
-						BlockHeight: data.BlockHeight,
-					}
-				}
-			}
-
+			// if p.cbChannel != nil {
```

Comment:
I think with this change to `metamorph` this channel will not be needed anymore, which means we can probably remove this whole package here: https://github.com/bitcoin-sv/arc/blob/1d668bdca5aafbc64fbe2a67b5dbd69e687abc22/asynccaller

Reply:
I will always vote for removing something 🕺